examples/simple_autopilot_private/main.tf

```diff
@@ -34,7 +34,7 @@ provider "kubernetes" {
 }
 
 module "gke" {
-  source                          = "../../modules/beta-autopilot-private-cluster/"
+  source                          = "github.com/terraform-google-modules/terraform-google-kubernetes-engine//modules/beta-autopilot-private-cluster/"
```

Comment:
This can continue to be `../../modules/beta-autopilot-private-cluster` as we use this for testing the module